src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -76,7 +78,7 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
         </div>
         <div className={`${styles.bottomSpace}`}>
           <b>Mean Difference</b>: {deltaPercent}%{' '}
-          {newIsBetter ? 'better' : 'worse'} ({delta})
+          {newIsBetter ? 'better' : 'worse'} ({delta} {baseUnit || newUnit})
```

Comment:
When there's no unit, this gives an extra space, I believe it should'nt be there in that case.
I think you can do something like this:
```
const unit = baseUnit || newUnit;
const deltaUnit = unit ? ` ${unit}` : '';

// Later
{newIsBetter ? 'better' : 'worse'} ({delta}{deltaUnit})